tests/dualtor/test_tor_ecn.py

```diff
@@ -304,17 +325,19 @@ def test_dscp_to_queue_during_encap_on_standby(
        testutils.send(ptfadapter, int(ptf_t1_intf.strip("eth")), non_encapsulated_packet, count=10)
 
 def test_ecn_during_decap_on_active(
-    apply_active_state_to_orchagent,
-    build_encapsulated_ip_packet,
-    rand_selected_interface, 
-    ptfadapter,
-    tbinfo, 
-    rand_selected_dut, 
-    tunnel_traffic_monitor
+    setup_dualtor_tor_active,
+    build_encapsulated_ip_packet, request,
+    rand_selected_interface, ptfadapter,
+    tbinfo, rand_selected_dut, tunnel_traffic_monitor
 ):
     """
     Test if the ECN stamping on inner header is matching with outer during decap on active
     """
+    if is_t0_mocked_dualtor(tbinfo):
+        request.getfixturevalue('apply_active_state_to_orchagent')
+    else:
+        request.getfixturevalue('toggle_all_simulator_ports_to_rand_selected_tor')
+
```

Comment:
These lines should be redundant now that you have `setup_dualtor_tor_active`

Reply:
removed